isthmus/src/test/java/io/substrait/isthmus/ComparisonFunctionTest.java

```diff
@@ -0,0 +1,29 @@
+package io.substrait.isthmus;
+
+import java.io.*;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ComparisonFunctionTest extends PlanTestBase {
```

Comment:
These are somewhat cursed to test via SQL roundtrip, because Calcite is too smart sometimes.

1. If we want to test Substrait <-> Calcite trips (which is what the function mappings are for), that can be achieved by building either the Substrait or Calcite plan trees directly, and then checking for that roundtrip.

2. If we actually want to test the SQL -> Calcite -> Substrait -> Calcite trip, we'll need to investigate configuring Calcite to not optimised/convert those functions away.

I don't think either of these are worth spending too much time though.

Reply:
Thanks for the review!  I'll remove these tests for now and just have the mapping in this PR in order to get them into Isthmus.  We can look more into adding tests in a subsequent PR.